projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
+
```

Comment:
What caused this newline addition in the generated files?

Reply:
I don't know, its generated. This happened when I ran `make generate` from the root of the project